helpers.py

```diff
@@ -64,23 +64,23 @@ def _pages_this_day():
 
     # Grab each issue that matches, and create a page_info structure for the
     # first page of that issue.  Grabbing only the set number requested by user
```

Comment:
The code here looks great, but this comment should be updated to reflect that issues are no longer retrieved :+1:

Reply:
Fixed!